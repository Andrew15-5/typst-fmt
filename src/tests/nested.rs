use super::*;

make_test!(code_func, "#{f(1,2,3)}");

make_test!(
    code_func_break,
    "#{f(1,2,3)}",
    Config {
        max_line_length: 2,
        ..Default::default()
    }
);


make_test!(
    code_func_break_nested,
    "#{{f(1,2,3)}}",
    Config {
        max_line_length: 2,
        ..Default::default()
    }
);