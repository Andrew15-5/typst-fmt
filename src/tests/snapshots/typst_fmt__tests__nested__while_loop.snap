---
source: src/tests/nested.rs
description: "INPUT\n===\n\"#let i = 0\\n#while true {\\n  i += 1\\n  if i > 15 { break }\\n  repr(i) + \\\" \\\"\\n}\"\n===\n#let i = 0\n#while true {\n  i += 1\n  if i > 15 { break }\n  repr(i) + \" \"\n}\n===\nFORMATTED\n===\n#let i = 0\n#while true {\n  i += 1\n  if i > 15 { break }\n  repr(i) + \" \"\n}"
expression: formatted
---
"#let i = 0\n#while true {\n  i += 1\n  if i > 15 { break }\n  repr(i) + \" \"\n}"
